password.py

```diff
@@ -1,49 +1,302 @@
 # Password Creator/Manager
 
+import random
+import shane_char 
+import re
+
+account_dict = {}
+account_info_dict = {}
 
 #<---- FIRST MAJOR FUNCTION ---->
 # Creates an account
 def account_create():
 
-    return
+    print("-"*18, "\nAccount Creation!"), print("-"*18)
+
+    while True:
+        username = input("Enter a Username:\n")
+        if len(username) < 5:
+            print("Username has to be 5 characters or more.")
+            continue
+        else:
+            account_info_dict["Username"] = username
+            break
+
+    if password_randomizer_query() == 'no':
+        while True:
+            password = input("Enter a Password:\n")
+            if len(password) < 6:
+                print("Password has to be 6 characters or more.")
+                continue
+            else:
+                account_info_dict["Password"] = password
+                break
+    
+    account_dict[username] = account_info_dict
+
+    security_questions()
+
+    account_save()
+
+    return 
+
+# Asks user to pick a question and saves the answer
+def security_questions():
+     sec_questions_dict = {
+         "1":"What's your pet's name",
+         "2":"What's your mother's middle name",
+         "3":"What's your best friend's nickname",
+         "4":"Which hospital were you born in"
+      }
+
+     print("-"*26)
+     for key,val in sec_questions_dict.items():
+         print(key,":",val)
+     print("-"*26)
+
+     security_choice = input("Please choose a security question (Type in corresponding number):\n")
+
+     value = sec_questions_dict[security_choice]
```

Comment:
We never check if the security choice is within the available choices of the dictionary, so if a user chooses 5 he/she will end up with an empty question.

Reply:
Wow thanks, totally missed that entirely.